local_dev/run_e2e_tests.sh

```diff
@@ -3,8 +3,15 @@
 # Abort if any of the following commands fails or variables are undefined
 set -eu
 
-DOCKER_REGISTRY=${DOCKER_REGISTRY:-"localhost"}
-ILLUMINATIO_IMAGE="${DOCKER_REGISTRY}:5000/illuminatio-runner:dev"
+# Default DOCKER_REGISTRY to the docker bind port if using docker driver
```

Comment:
It's not only the port that will be returned rather the address + port for the docker registry binding.